packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -259,11 +329,12 @@ class SemanticsData extends Diagnosticable {
         && typedOther.scrollPosition == scrollPosition
         && typedOther.scrollExtentMax == scrollExtentMax
         && typedOther.scrollExtentMin == scrollExtentMin
-        && typedOther.transform == transform;
+        && typedOther.transform == transform
+        && setEquals(typedOther.customAccessibilityActions, customAccessibilityActions);
```

Comment:
yikes, this is one expensive `==` method

Reply:
I _think_  replacing the set of custom actions with a sorted list of ints will make this faster...